### PR TITLE
Feat: create the balance sheet screen

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Create the balance sheet screen](https://trello.com/c/LnaeTGqg/863-create-the-balance-sheet-screen)

File: new-lamassu-admin/src/pages/Assets/Assets.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      direction: 'in'
+    },
+    {
+      id: 'unhedgedWalletAssets',
+      display: 'Unhedged wallet assets',
+      amount: 3978,
+      currency: 'USD',
+      direction: 'in'
+    }
+  ]
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Assets/Assets.js]: You can move this outside the component. Hmm, if you're kind of simulating the query response and want to confine this data to the component's lifespan, I'm not sure what the correct procedure is. Better confirm with Rafael on this one

File: new-lamassu-admin/src/pages/Assets/Assets.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+  const filterByClass = x =>
+    R.filter(it => R.path(['class'])(it) === x)(mockData)
+
+  return (
+    <>
+      <TitleSection title="Balance sheet" />
+      <div className={classes.root}>
+        <Grid container>
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Assets/Assets.js:174]: Isn't this Grid element redundant? As you already have the wrapping div

File: new-lamassu-admin/src/pages/Assets/Assets.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+const HeaderCell = withStyles({
+  root: {
+    borderBottom: '4px solid white',
+    padding: 0,
+    paddingLeft: 20,
+    paddingRight: 20,
+    backgroundColor: 'white'
+  }
+})(TableCell)
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Assets/Assets.js]: You can move up the object containing
```
{
  borderBottom: '4px solid white',
  padding: 0,
  paddingLeft: 20,
  paddingRight: 20
}
```

And do something like this:
```suggestion
const Cell = withStyles({
  root: refactoredObj
})(TableCell)

const HeaderCell = withStyles({
  root: {
    ...refactoredObj
    backgroundColor: 'white'
  }
})(TableCell)
```

File: new-lamassu-admin/src/pages/Assets/Assets.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                      <Cell align="right">
+                        <P>{`${selectAmountPrefix(asset)}
+                         ${formatCurrency(Math.abs(asset.amount))} ${
+                          asset.currency
+                        }`}</P>
+                      </Cell>
+                    </TableRow>
+                  )
+                }
+                return null
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Assets/Assets.js]: I think the reverse is more readable. Pardon if formatting goes wrong :D
```suggestion
                if (!(idx < numToRender)) return <></>
                return (
                  <TableRow className={classes.row} key={idx}>
                    <Cell align="left">
                      <P>{asset.display}</P>
                    </Cell>
                    <Cell align="right">
                      <P>{`${selectAmountPrefix(asset)}
                       ${formatCurrency(Math.abs(asset.amount))} ${
                        asset.currency
                      }`}</P>
                    </Cell>
                  </TableRow>
                )
```

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Assets/Assets.js (hunk 2 above)
Comment (josepfo) [reply new-lamassu-admin/src/pages/Assets/Assets.js:174]: Hmm, I think not, if removed the tables will be displayed in a single column.

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**